optaplanner-examples/src/main/java/org/optaplanner/examples/tsp/app/TspApp.java

```diff
@@ -66,8 +71,13 @@ protected TspPanel createSolutionPanel() {
     }
 
     @Override
-    protected AbstractSolutionExporter createSolutionExporter() {
-        return new TspExporter();
+    protected Set<AbstractSolutionExporter> createSolutionExporters() {
+        Set<AbstractSolutionExporter> exporters = new HashSet<>();
+        exporters.add(new TspExporter());
+        exporters.add(new SvgTspExporter());
+        exporters.add(new SvgTspExporter2());
```

Comment:
Let's only have 1 SVG exporter.
I understand that you may have some specific use case that requires two slightly different SVG, but I'd rather not expose that in a generic example app.

Reply:
I don't think this is good, as there is no (simple) way of parametrizing one exporter to do different flavours. And as you can see / guess from the rendered SVGs, there are specific use cases.
It may be possible to use an Accessory view, but I think that is way above simple usages.
I did on purpose use two different file suffixes for the different exporters for these use cases.

Reply:
OK, if we can give the different exporter classes some descriptive names (as opposed to `1` and `2`), so be it.